modules/affinity/src/main/java/org/apache/ignite/internal/affinity/ditributed/AffinityManager.java

```diff
@@ -120,7 +120,7 @@ private void subscribeToCalculateAssignment(long startRevision) {
 
         String tableInternalPrefix = INTERNAL_PREFIX + "#.assignment";
```

Comment:
Change the string with placeholder to string with prefix.
Because you explicitly invoke a method which required prefix.